src/server/routes/entity/edition.js

```diff
@@ -166,6 +167,7 @@ router.get('/:bbid/edit', auth.isAuthenticated, loadIdentifierTypes,
 			title: 'Edit Edition',
 			heading: 'Edit Edition',
 			subheading: 'Edit an existing Edition in BookBrainz',
+			script: 'edition',
```

Comment:
Also need to update the template name from `entity/create/<entity>` to `entity/create/create-common` for all entity edit routes.